package.json

```diff
@@ -77,5 +81,13 @@
     "ecdsa-sig-formatter": "^1.0.11",
     "elliptic": "^6.4.1",
     "sha.js": "^2.4.11"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "release": {
+    "extends": "./.github/.releaserc.json"
   }
 }
```

Comment:
I think the `husky` [config is missing](https://github.com/hirosystems/gh-actions-example/blob/main/package.json#L12). Without it, I'm not sure the commit hook will work properly.

Reply:
[husky sort-of moved away from the package.json config](https://blog.typicode.com/husky-git-hooks-javascript-config/)